docs/configs/bebop.py

```diff
@@ -0,0 +1,42 @@
+from parsl.addresses import address_by_hostname
+from parsl.launchers import SrunLauncher
+from parsl.providers import SlurmProvider
+
+from funcx_endpoint.endpoint.utils.config import Config
+from funcx_endpoint.executors import HighThroughputExecutor
+
+# fmt: off
+
+# PLEASE UPDATE user_opts BEFORE USE
+user_opts = {
+    'bebop': {
+        'worker_init': '',
+        'scheduler_options': '',
+        'partition': 'bdws',
+    }
+}
```

Comment:
Would it be helpful to inject an intentional breakage, so as draw folks copy-and-paste happy attention immediately to the spot?  Something like:
```python
user_opts = {
    ...
}
# Edit above, and remove the following `raise` line
raise NotImplemented("PLEASE EDIT THE DEFAULT CONFIGURATION FILE")
```

Reply:
Good question. We haven't anywhere else and this config would technically work if the user has installed funcx-endpoint within their default python env.

Reply:
I like this idea!

Reply:
Sounds good. I'll make a new story to add this to each example!